### PR TITLE
update dependencies/handle CVEs

### LINKED ISSUE #15: Use system properties to set DEBUG_ADDRESS and LOG_LEVEL
[IntegrationTestSetup.java](https://github.com/exasol/azure-blob-storage-document-files-virtual-schema/blob/ced87a7b501415d1b3171d8d6d81a9ec73406df6/src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java) sets constant values for `DEBUG_ADDRESS` and `LOG_LEVEL`. 

```java
if (System.getProperty("test.vs-logs", "false").equals("true")) {
   properties.put("DEBUG_ADDRESS", "127.0.0.1:3001");
```

Please consider to remove these lines and propose using the system properties supported by https://github.com/exasol/test-db-builder-java/issues/103 in the documentation (either user_guide or developer_guide).

### LINKED ISSUE #19: Review and fix vulnerabilities

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #19 
Closes #15

### CODE REVIEW

File: doc/changes/changes_1.1.3.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Virtual Schema for document data in files on Azure Blob Storage 1.1.3, released 2023-02-07
+
+Code name: Reviewed and fixed vulnerabilities, updated dependencies and refactored integration test setup class.
+
+## Summary
+
+Reviewed and fixed vulnerabilities. 
+Refactored IntegrationTestSetup to use system properties.
+## Bugfixes / Vulnerabilities
```
Comment (ckunki) [inline doc/changes/changes_1.1.3.md:9]:
```suggestion

## Bugfixes / Vulnerabilities
```

File: pom.xml [hunk 2]
```diff
@@ -208,7 +245,8 @@
                 <artifactId>maven-assembly-plugin</artifactId>
                 <groupId>org.apache.maven.plugins</groupId>
                 <configuration>
-                    <finalName>document-files-virtual-schema-dist-${vs-common-document-files.version}-azure-blob-storage-${project.version}</finalName>
+                    <finalName>
+                        document-files-virtual-schema-dist-${vs-common-document-files.version}-azure-blob-storage-${project.version}</finalName>
```
Comment (ckunki) [inline pom.xml:249]:
```suggestion
                    <finalName>document-files-virtual-schema-dist-${vs-common-document-files.version}-azure-blob-storage-${project.version}</finalName>
```

File: src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java [hunk 3]
```diff
@@ -172,10 +178,6 @@ private Map<String, String> getVirtualSchemaProperties(final String mapping) {
         if (!debugProperty.isBlank() || !profileProperty.isBlank()) {
             properties.put("MAX_PARALLEL_UDFS", "1");
         }
-        if (System.getProperty("test.vs-logs", "false").equals("true")) {
-            properties.put("DEBUG_ADDRESS", "127.0.0.1:3001");
-            properties.put("LOG_LEVEL", "ALL");
-        }
```
Comment (ckunki) [inline src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java:178]: 👍

File: src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java [hunk 4]
```diff
@@ -84,25 +84,31 @@ private ConnectionDefinition createConnectionDefinition() {
         final JsonObjectBuilder configJson = getConnectionConfig();
         return createConnectionDefinition(configJson);
     }
+
     private Optional<String> getHostOverride() {
-        return this.absTestSetup.getHostOverride().map(address -> this.exasolTestSetup
-                .makeTcpServiceAccessibleFromDatabase(ServiceAddress.parse(address)).toString());
+        return this.absTestSetup.getHostOverride().map(address -> {
```
Comment (ckunki) [inline src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java:89]: If I had a wish, then I'd propose to try to remove the use of `Optional` in `AbsTestSetup.getHostOverride()` and `IntegrationTestSetup.getHostOverride()`. But I do not want to block your release and leave it up to you, as this change can also be done in a separate PR at a later point in time.

**Review by ckunki — CHANGES_REQUESTED** [review]

File: src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java (hunk 4 above)
Comment (pj-spoelders) [reply src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java:89]: Let's keep that for a later time :) I want to get this out of the door today. Save your wish :)

Comment (ckunki) [reply src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java:89]: 👍

Comment (ckunki) [reply src/test/java/com/exasol/adapter/document/files/IntegrationTestSetup.java:89]: https://github.com/exasol/azure-blob-storage-document-files-virtual-schema/issues/20

**Review by ckunki — APPROVED** [review]

**Pull request merged**